content/07.pathogenesis.md

```diff
@@ -53,7 +53,7 @@ A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:978069111
 The effective reproduction number, R~t~, describes how the reproduction number may change over time, and is used to quantify deviations in R from R~0~ that would occur, for example, as some fraction of the population became infected or as interventions were put into place.
 When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
 R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
-Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1126/science.272.5258.74].
+Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@doi:10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
```

Comment:
```suggestion
Modeling approaches are typically based upon a classic epidemiological model structure: the susceptible-infected-recovered (SIR) model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
```

Good catch on the wrong year @alavendelm.  If we break convention and cite with the URL here, Zotero is able to extract the correct metadata:
```
$ manubot cite --render doi:10.1098/rspa.1927.0118
1. A contribution to the mathematical theory of epidemics
Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character
(1997-01) https://doi.org/fwx2qw
DOI: 10.1098/rspa.1927.0118

$ manubot cite --render https://doi.org/10.1098/rspa.1927.0118
1. A contribution to the mathematical theory of epidemics
William Ogilvy Kermack, A. G. McKendrick, Gilbert Thomas Walker
Proceedings of the Royal Society of London. Series A, Containing Papers of a Mathematical and Physical Character (1927-08-01) https://royalsocietypublishing.org/doi/10.1098/rspa.1927.0118
DOI: 10.1098/rspa.1927.0118
```
Should Gilbert Thomas Walker be listed as an author or not?  The HTML source lists him as an author, which is what Zotero picks up.  The PDF says "communicated by" Walker and has only two authors.

Reply:
No Walker is not an author - "communicated by" him (I guess like PNAS direct submission, back in the day). Btw paper was first published online in 01/1997 so I guess that's what confused Zotero.

Reply:
Got it. I'll make a commit in your fork to update the manual references so that Walker isn't included.